fix: Rollback admin/workspace route to admin/info (#30728)
RocketChat/Rocket.Chat: .changeset/quick-cups-sparkle.md

---
"@rocket.chat/meteor": patch
---

> Change `/admin/workspace` path to `/admin/info` path as it was before 

Reviewer: ```suggestion
Rolled back a change of route from `/admin/workspace` to `/admin/info`
```

Suggestion so the message aligns with the guidelines for changesets https://developer.rocket.chat/contribute-to-rocket.chat/modes-of-contribution/participate-in-rocket.chat-development/development-workflow#4.-adding-changeset-to-your-pull-request

Author: I don't this that Rolled back it's a good term to use, since it's a technical term

Reviewer: That's a good point, please feel free to suggest something different
